### ISSUE TITLE
数据集问题

### ISSUE THREAD

**Issue by YYDS-cc** [KyanChen/BuildingExtraction#10]
感谢分享代码！有个关于数据集的问题想问一下。
刚刚接触建筑物提取，文中提到的INRIA数据集，请问用于实验的文件夹格式是什么样子的啊，我下载的数据集包含test和train两个文件夹，只在train文件夹里有gt文件夹，请问是将Cutimgsegwithlabel中的parent_path直接替换成上述我说的train文件夹的路径就可以了吗？如果按照这个路径来，得不到分割结果欸，麻烦大佬给解答下，万分感谢！

**Comment by KyanChen** [comment]
您好，我们在论文中给出了相关说明“Since the authors did not release the label for the test set, we divide the original training set into a training set, a validation set, and a test set with the ratio of 6:2:2. When we train our model, we crop all training images to a set of 512×512 image slices with an overlap ratio of 0.9.”
需要先把train文件夹的图像按6:2:2分成a training set, a validation set, and a test set，然后再使用Cutimgsegwithlabel脚本切割成512的大小

**Comment by YYDS-cc** [comment]
哇，回复的真快！
明白你的意思了，感谢大佬！

**Comment by ShengqiangX** [comment]
作者您好，感谢您的分享，请问一下WHU数据集网址打不开，，然后我下载的其他数据是tiff格式，怎么去处理呀
